tests/pauli/dla/test_center.py

```diff
@@ -60,19 +61,30 @@ def test_center_pauli_word_pauli_True(pauli):
         qml.pauli.PauliWord({0: "X"}),
         qml.pauli.PauliWord({0: "X", 1: "X"}),
         qml.pauli.PauliWord({1: "Y"}),
+        qml.pauli.PauliWord({0: "X", 1: "Z"}),
     ]
+    x0_pw = qml.pauli.PauliWord({0: "X"})
+    center = qml.center(ops, pauli=pauli)
+    assert isinstance(center, list)
+    assert len(center) == 1
     if pauli:
-        assert qml.center(ops, pauli=pauli) == [qml.pauli.PauliWord({0: "X"})]
+        assert isinstance(center[0], qml.pauli.PauliSentence) and len(center[0]) == 1
+        assert x0_pw in center[0]
+        assert np.isclose(center[0][x0_pw], -1)
     else:
-        assert qml.center(ops, pauli=pauli) == [qml.X(0)]
+        assert isinstance(center[0], qml.ops.op_math.SProd)
+        assert qml.equal(center[0], -1 * qml.X(0))
```

Comment:
New numerical prefactor makes comparison a bit more complex.

Reply:
good to know that `qml.equal` works for this 👍

Reply:
Agreed! Caveat: Unfortunately does not work for `PauliSentence` yet.